src/dialogs/more-info/controls/more-info-climate.js

```diff
@@ -227,7 +227,9 @@ class MoreInfoClimate extends LocalizeMixin(EventsMixin(PolymerElement)) {
                   items="[[stateObj.attributes.fan_list]]"
                   on-dom-change="handleFanListUpdate"
                 >
-                  <paper-item>[[item]]</paper-item>
+                  <paper-item
+                    >[[_localizeOperationMode(localize, item)]]</paper-item
```

Comment:
You're not calling the `_localizeFanMode` function. Did you test this?

Reply:
Apologies, honestly not sure how that got lost during my local merge & rebase.  Tested and working.

Reply:
![thermos](https://user-images.githubusercontent.com/18057952/52976758-4ce69d80-3390-11e9-811b-979640d847fe.PNG)